### PR TITLE
トップ画面に遷移するコントローラを実装

### PR DESCRIPTION
**Pull request by tkeshun**

### CODE REVIEW

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.example.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@RequestMapping("/show-top-page")
+public class ShowTopPage {
+
+	/**
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	 * 注目確認画面を表示します.
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	 *
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/show-top-page")
+public class ShowTopPage {
+
+	/**
+	 * 注目確認画面を表示します.
+	 * 
+	 * @param orderId 注文の主キー
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused @param tag for 'orderId'.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+	 * @return 注文確認画面
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+	 */
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+	@GetMapping("/top")
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+	public String showTopPage() {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+		return "ec/top";
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:20]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:20]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/common/SecurityConfig.java [hunk 12]
```diff
@@ -42,7 +42,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		http.authorizeHttpRequests(ahr -> ahr
 				.requestMatchers("/login-user/to-login", "/register-user/to-register", "/login-user/login-user",
 						"/register-user/register-user", "/show-item/show-item-list", "/shopping-cart/show-cart",
-						"/show-detail/show-item-detail")
+						"/show-detail/show-item-detail", "/show-top-page/top")
```
Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).

Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**